File: app/src/main/java/com/example/spotibae/Models/Song.java
package com.example.spotibae.Models;

import java.util.List;

public class Song {
    public String id;
    public String name;
    public String artistName;
    public String href;
    public String uri;

    public Song() { }

    public Song(String id, String name, String artistName, String href, String uri) {
        this.id = id;
        this.artistName = artistName;
        this.name = name;
        this.href = href;
        this.uri = uri;
    }

    @Override
    public String toString() {
        return "Song: " + "ID: " + id + ", Artist Name: " + artistName + ", Album Name: " + name + ", Href: " + href + ", URI: " + uri + "\n";
    }
}

File: app/src/main/java/com/example/spotibae/Models/Song.kt
package com.example.spotibae.Models

class Song {
    lateinit var id: String
    lateinit var name: String
    lateinit var artistName: String
    lateinit var href: String
    lateinit var uri: String

    constructor() {}
    constructor(id: String, name: String, artistName: String, href: String, uri: String) {
        this.id = id
        this.artistName = artistName
        this.name = name
        this.href = href
        this.uri = uri
    }

    override fun toString(): String {
        return "Song: ID: $id, Artist Name: $artistName, Album Name: $name, Href: $href, URI: $uri\n"
    }
}